en/components/general/update-guide.md

```diff
@@ -51,6 +51,35 @@ Unfortunately not all changes can be automatically updated. Changes below are sp
 
 For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from the "From 6.x .." section apply those changes and work your way up:
 
+## From 14.1.x to 14.2.x
+### General
+- **Breaking Change** - All **igx** prefixes are replaced with **ig** prefixes. For example, **igx-**typography is now **ig-**typography.
+
+- Add this line in your `angular.json` file:
+```
+"stylePreprocessorOptions": {
+    "includePaths": ["node_modules"]
+}
+```
+to access the `igniteui-theming package`, without writing the full path. You should have the igniteui-theming package installed in your `node_modules` folder first. If it's not installed automatically, use the `npm install igniteui-theming` command in your project directory.
+
+### Themes
+- **Breaking Change** - All `$grays` variable names are switched with `$gray`. <br />
```

Comment:
This should be more specific. The name has changed in several places, not just the argument `$grays`. So mention that the $grays input argument must be changed in the `palette`, `color`, and `contrast-color` functions, but also that `--igx-grays-*` now is `--ig-gray-*`. All this will be much better explained using code samples to support the text description.